src/main/java/edu/harvard/iq/dataverse/engine/command/impl/ListDataverseContentCommand.java

```diff
@@ -32,11 +34,14 @@ public ListDataverseContentCommand(DataverseRequest aRequest, Dataverse anAffect
 
     @Override
     public List<DvObject> execute(CommandContext ctxt) throws CommandException {
+        List<DvObject> availableObjects = new ArrayList();
         if (getRequest().getUser().isSuperuser()) {
-            return ctxt.dvObjects().findByOwnerId(dvToList.getId());
+            availableObjects = ctxt.dvObjects().findByOwnerId(dvToList.getId());
         } else {
-            return ctxt.permissions().whichChildrenHasPermissionsForOrReleased(getRequest(), dvToList, EnumSet.of(Permission.ViewUnpublishedDataverse, Permission.ViewUnpublishedDataset));
+            availableObjects =  ctxt.permissions().whichChildrenHasPermissionsForOrReleased(getRequest(), dvToList, EnumSet.of(Permission.ViewUnpublishedDataverse, Permission.ViewUnpublishedDataset));
         }
+        availableObjects.sort(Comparator.comparing(DvObject::getDisplayName));
```

Comment:
Since we don't display the name of datasets, wouldn't it make more sense to sort by id?

Reply:
And if sorting by id, can we just do that in the db query? (seems better performance that sorting after in the java. We could also sort first by dtype, then by id if we wanted to effectively group the dataverses, then datasets)

Reply:
Is there agreement that we should just use id? (just use order by in the query and ignore type)

Reply:
My vote is first by dtype, then id, which can both be done in the query.

Reply:
I updated the query to order by dType (descending to list dataverses first) then dvObject ID. I also updated the doc to note this ordering.